src/main/webapp/app/shared/metis/forwarded-message/forwarded-message.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="forwarded-message-container">
+    <div class="left-border-line"></div>
+    <div class="forwarded-message-content">
+        <span class="forwarded-message-channel fs-x-small d-flex align-items-center">
+            <fa-icon [icon]="faShare" class="item-icon" style="margin-right: 0.3rem"></fa-icon>
+            {{ 'artemisApp.metis.forward.forwardedFrom' | artemisTranslate }} {{ sourceName }}
+            @if (viewButtonVisible) {
+                <button
+                    type="button"
+                    class="btn btn-link p-0"
+                    style="width: fit-content; font-size: 0.7rem; margin-left: 0.2rem"
+                    (click)="onTriggerNavigateToPost()"
+                    jhiTranslate="artemisApp.metis.forward.viewConversation"
+                ></button>
+            }
+        </span>
+        <div class="forwarded-message-header">
+            <jhi-profile-picture
+                imageSizeInRem="1.5"
+                fontSizeInRem="0.6"
+                imageId="user-profile-picture"
+                defaultPictureId="user-default-profile-picture"
+                [authorId]="originalPostDetails()?.author?.id"
+                [authorName]="originalPostDetails()?.author?.name"
+                [imageUrl]="originalPostDetails()?.author?.imageUrl"
+                style="margin-right: 0.2rem"
+            >
+            </jhi-profile-picture>
+            <span class="forwarded-message-author">{{ originalPostDetails()?.author?.name }}</span>
+            <span class="post-header-date-separator">-</span>
+            <span class="post-header-date">
+                @if (postingIsOfToday) {
+                    <span [jhiTranslate]="todayFlag ?? ''" id="today-flag" class="fs-x-small"></span>
+                }
+                <span class="fs-x-small" [disableTooltip]="postingIsOfToday" ngbTooltip="{{ originalPostDetails()?.creationDate | artemisDate: 'time' }}">
+                    {{ postingIsOfToday ? (originalPostDetails()?.creationDate | artemisDate: 'time') : (originalPostDetails()?.creationDate | artemisDate: 'short-date') }}
+                </span>
+            </span>
+        </div>
+        <div class="forwarded-message-body" #messageContent [ngClass]="{ expanded: showFullForwardedMessage }">
+            <div class="preserve-line-breaks">{{ displayedForwardedContent() }}</div>
```

Comment:
I think you need to use the `jhi-posting-content` component here, otherwise it wont render properly.

<img width="648" alt="Bildschirmfoto 2024-12-10 um 13 11 42" src="https://github.com/user-attachments/assets/4aab1b5f-ad4d-4974-9882-c7ee64c14021">